### DISCUSSION TITLE
Default tool on taking screenshot

### DISCUSSION THREAD

**Discussion by supermikes** [flameshot-org/flameshot#2863] [Ideas]
Hi, 

Any possibility to set a default tool after taking screenshot? e.g. I take screenshot and then marker tool is already selected. 

Thanks.

**Comment by mmahmoudian** [comment]
No such feature doesn't exist. You can instead use keyboard shortcuts to activate tools.

**Reply by supermikes** [reply]
Will this be possible to implement one day in future?

**Reply by mmahmoudian** [reply]
> Will this be possible to implement one day in future?

It can certainly be discussed if you [create a Feature Request](https://github.com/flameshot-org/flameshot/issues/new?assignees=&labels=Enhancement&template=feature-request.yaml) about it and write your justification on why this is a good addition to Flameshot. I'm not against it, but at this point I'm not sold on the idea either. But I don't represent the whole dev team and I can just state my opinion. Therefore, in the feature request please try to include:
- why this is a good feature to have
- how you think the configuration should be

A well explained idea is always easier to discuss and get to a conclusion. 

Also, if you are willing to implement it and create a Pull Request, please discuss it first in a feature request so that all devs have the opportunity to express their opinions.

**Reply by supermikes** [reply]
Thanks - I have thought about this and decided to create a macro instead - this resolves my issue so suggestion won't be needed. Thanks for your responses :)
